docs/settings.md

```diff
@@ -44,3 +44,50 @@
+| `hosted-signaling-server-account`    | array                                                            | `{}`       | 🖌️    | Account information of the hosted signaling server                                                                                                                               |
+| `stun_servers`                       | array[]                                                          | `[]`       | 🖌💻️  | List of STUN servers, should be configured via the web interface or the OCC commands                                                                                             |
+| `turn_servers`                       | array[]                                                          | `[]`       | 🖌️💻  | List of TURN servers, should be configured via the web interface or the OCC commands                                                                                             |
+| `signaling_servers`                  | array[]                                                          | `[]`       | 🖌️💻  | List of signaling servers, should be configured via the web interface or the OCC commands                                                                                        |
+| `signaling_mode`                     | string<br>`internal` or `external` or `conversation_cluster`     | `internal` |        | `internal` when no HPB is configured, `external` when configured, `conversation_cluster` is an experimental flag that is deprecated                                              |
+| `sip_bridge_dialin_info`             | string                                                           |            | 🖌️    | Additional information added in the SIP dial-in invitation mail and sidebar                                                                                                      |
+| `sip_bridge_shared_secret`           | string                                                           |            | 🖌️    | Shared secret allowing the SIP bridge to authenticate on the Nextcloud server                                                                                                    |
+| `signaling_ticket_secret`            | string                                                           |            |        | Secret used to secure the signaling tickets for guests (255 character random string)                                                                                             |
+| `signaling_token_alg`                | string<br>`ES256`, `ES384`, `RS256`, `RS384`, `RS512` or `EdDSA` | `ES256`    |        | Algorithm for the signaling tickets,                                                                                                                                             |
```

Comment:
Is something missing after the comma?

Reply:
no, just a stray comma. Going to remove it.